internal/params/config.go

```diff
@@ -37,7 +37,7 @@ var (
 	TestnetChainConfig = &ChainConfig{
 		ChainID:         TestnetChainID,
 		CrossTxEpoch:    big.NewInt(0),
-		CrossLinkEpoch:  big.NewInt(3),
+		CrossLinkEpoch:  big.NewInt(4),
 		StakingEpoch:    big.NewInt(4),
```

Comment:
Should we have shorter PreStaking and Staking Epoch gap? With this configuration, we have to wait for ~1 hour for staking logic to kick in? May be have similar configuration as localnet?

Reply:
testnet block per epoch is 75, so about 30 min